ADBench/run-all.ps1

```diff
@@ -51,7 +51,10 @@ param(# Which build to test.
       [double]$time_limit=10,  
       
       # Kill the test after this many seconds 
-      [double]$timeout=300,    
+      [double]$timeout=300,
+
+      # Kill the test if it consumes more than this many gigabytes of RAM
+      [double]$max_memory_amount=8,
```

Comment:
I would suggest calling this `max_memory_amount_in_gb` so that it's clearer what the units are.

Reply:
Done